src/celttf/truetypefont.cpp

```diff
@@ -495,8 +495,8 @@ TextureFontPrivate::flush()
     glBufferData(GL_ARRAY_BUFFER, sizeof(FontVertex) * MaxVertices, nullptr, GL_STREAM_DRAW);
     glBufferData(GL_ELEMENT_ARRAY_BUFFER, sizeof(GLushort) * MaxIndices, nullptr, GL_STREAM_DRAW);
 
-    glBufferSubData(GL_ARRAY_BUFFER, 0, sizeof(FontVertex) * m_fontVertices.size(), m_fontVertices.data());
-    glBufferSubData(GL_ELEMENT_ARRAY_BUFFER, 0, sizeof(GLushort) * indexes.size(), indexes.data());
+    glBufferData(GL_ARRAY_BUFFER, sizeof(FontVertex) * m_fontVertices.size(), m_fontVertices.data(), GL_STREAM_DRAW);
```

Comment:
Do we still need the previous `glBufferData` calls on lines 495-496?

Reply:
yes, but with a proper size, lem me fix this. (passing data==nullptr and the same size and usage as earlier means "i don't need this data, if the buffer is in use, then allocate a new one").